test/black-box/iteminformation/ItemInformationGetCommand.test.ts

```diff
@@ -6,51 +6,65 @@ import { rpc, api } from '../lib/api';
 import { BlackBoxTestUtil } from '../lib/BlackBoxTestUtil';
 import { Commands } from '../../../src/api/commands/CommandEnumType';
 import { CreatableModel } from '../../../src/api/enums/CreatableModel';
+import { GenerateListingItemTemplateParams } from '../../../src/api/requests/params/GenerateListingItemTemplateParams';
 
 describe('ItemInformationGetCommand', () => {
     const testUtil = new BlackBoxTestUtil();
     const method = Commands.ITEMINFORMATION_ROOT.commandName;
     const subCommand = Commands.ITEMINFORMATION_GET.commandName;
 
```

Comment:
rename these too so that the commands are more readable, so you don't have to guess and check what rpc call we are making when you see: testUtil.rpc(command, subcommand, asdf, asdf)

const itemInformationCommand = Commands.ITEMINFORMATION_ROOT.commandName;
const itemInformationGetCommand = Commands.ITEMINFORMATION_GET.commandName;